OSDOCS-16155: OCPSTRAT-2482 Advanced Audit Logging Framework GA (#106553)
openshift/openshift-docs: modules/spo-logging-correlate-02.adoc

> :_mod-docs-content-type: CONCEPT

Reviewer: this is a reference module

:_mod-docs-content-type: CONCEPT
[id="spo-logging-correlate-02_{context}"]
= Kubernetes API audit log output

> 

Reviewer 2: ```suggestion
```

[role="_abstract"]

> The output of the Kubernetes API audit log is well formatted YAML. It contains the SPO_EXEC_REQUEST_UID field which provides the correlation key with which to search the Advanced Audit Logging output. 

Reviewer 2: ```suggestion
The output of the Kubernetes API audit log is YAML. It contains the SPO_EXEC_REQUEST_UID field. This field provides the correlation key with which you can search the Advanced Audit Logging output. 
```
is this outputting just a YAML snippet?
